templates/systemd_pulp_workers

```diff
@@ -6,3 +6,11 @@ PULP_CONCURRENCY=<%= scope['pulp::num_workers'] %>
 
 # Configure Python's encoding for writing all logs, stdout and stderr
 PYTHONIOENCODING="UTF-8"
+
+# To avoid memory leaks, Pulp can terminate and replace a worker after processing X tasks. If
+# left commented, process recycling is disabled. PULP_MAX_TASKS_PER_CHILD must be > 0.
+<% if !scope['pulp::max_tasks_per_child'].nil? and scope['pulp::max_tasks_per_child'] > 0 %>
```

Comment:
Generally we do this kind of check:

```erb
<% unless [nil, :undefined, :undef, ''].include?(scope.lookupvar('pulp::max_tasks_per_child')) -%>
```

Some examples:
https://github.com/Katello/puppet-pulp/blob/master/templates/server.conf.erb#L415

The reason it's like this is because of how different puppet versions handle undefined variables, it's a bit of a mess that'll be fixed once we can go to Puppet 4 only.